app/controllers/cart/cart_items_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Cart::CartItemsController < ApplicationController
+  before_action :set_cart_item, only: %i[update]
+
+  def create
+    product = Product.purchasable.find(cart_item_params[:product_id])
+    current_cart.add_cart_item(product, cart_item_params[:quantity])
+    redirect_to cart_url, notice: '商品をカートに追加しました'
+  end
+
+  def update
+    quantity = cart_item_params[:quantity].to_i
+    if quantity == 0  # rubocop:disable all
```

Comment:
後から気づいたけど、ユーザー入力なので 0 未満とかきちんと弾かないといけないですね
雑にやるならこんな感じか
```suggestion
    quantity = [cart_item_params[:quantity].to_i, 0].max
    if quantity == 0  # rubocop:disable all
```

Reply:
[fa4c529](https://github.com/kk0000-kk/sakura-market/pull/25/commits/fa4c52980bd87f28e9e7a5dde34ed5aaded550a1)